core/state/statedb.go

```diff
@@ -87,13 +189,16 @@ type StateDB struct {
 	fullProcessed  bool
 	pipeCommit     bool
 
-	snaps          *snapshot.Tree
-	snap           snapshot.Snapshot
-	snapAccountMux sync.Mutex // Mutex for snap account access
-	snapStorageMux sync.Mutex // Mutex for snap storage access
-	snapDestructs  map[common.Address]struct{}
-	snapAccounts   map[common.Address][]byte
-	snapStorage    map[common.Address]map[string][]byte
+	snapMux           sync.Mutex
```

Comment:
rebase caused, `snaps` has been divided into `snapAccountMux` and `snapStorageMux`